src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -1293,14 +1300,16 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
   const errorCopy = useMemo(() => {
     // Order matters here. Since we're dealing with two assets potentially, we want to show the most relevant error message possible i.e
     // 1. Asset unsupported by wallet
-    // 2. pool halted
-    // 3. smart contract deposits disabled
-    // 4. pool asset balance
-    // 5. pool asset fee balance, since gas would usually be more expensive on the pool asset fee side vs. RUNE side
-    // 6. RUNE balance
-    // 7. RUNE fee balance
+    // 2. pool missing
+    // 3. pool halted
+    // 4. smart contract deposits disabled
+    // 5. pool asset balance
+    // 6. pool asset fee balance, since gas would usually be more expensive on the pool asset fee side vs. RUNE side
+    // 7. RUNE balance
+    // 8. RUNE fee balance
     // Not enough *pool* asset, but possibly enough *fee* asset
     if (!walletSupportsOpportunity) return translate('common.unsupportedNetwork')
+    if (poolMissing) return translate('pools.poolMissing')
```

Comment:
@woodenfurniture do we want e5b7262306b as its own PR, not to include LP diff here?

Reply:
goood call https://github.com/shapeshift/web/pull/6502

Reply:
aaaand reverted from this PR